src/protocols/abstract/Realtime.ts

```diff
@@ -59,6 +59,8 @@ export abstract class BaseProtocolRealtime extends KuzzleAbstractProtocol {
    * Called when the client's connection is closed
    */
   clientDisconnected () {
+    clearInterval(this.pingIntervalId);
```

Comment:
The Realtime object is a generic class that does not know anything about those properties, since they are specific to the websocket protocol